roles/awscommon/tasks/vpc-facts.yml

```diff
@@ -8,3 +8,62 @@
   register: vpc_instance_facts
   tags:
     - always
+
+- name: get VPC data
```

Comment:
We have other tasks called this. How about `get VPC net data`

Reply:
I'll rename